src/components/ChatEntry.js

```diff
@@ -1,22 +1,73 @@
 import React from 'react';
+// import {useState} from 'react';
+import TimeStamp from './TimeStamp'
 import './ChatEntry.css';
 import PropTypes from 'prop-types';
 
-const ChatEntry = (props) => {
-  return (
-    <div className="chat-entry local">
-      <h2 className="entry-name">Replace with name of sender</h2>
+const ChatEntry = ({id, sender, body, timeStamp, liked, updateLiked}) => {
+  //The original way I implemented getting the date, before realizing TimeStamp existed
+  // const newDate = new Date()
+  
+  // let diff = (newDate.getTime() - new Date(timeStamp).getTime()) / 1000;
+  // diff /= (60*60*24);
+  let timePassed = {
+    time: timeStamp
+  };
+  let currentYears = TimeStamp(timePassed)
+  
+  // const [buttonText, setButtonText] = useState('🤍')
+  const buttonText = liked ? '❤️' : '🤍'
+  const toggleLiked = (id, likedStatus) => {
+    updateLiked(id, likedStatus);
+  };
+  if (Number(id)%2){
+    return (
+      <div className="chat-entry local">
+      <li key={id}>
+      <h2 className="entry-name">{sender}</h2>
       <section className="entry-bubble">
-        <p>Replace with body of ChatEntry</p>
-        <p className="entry-time">Replace with TimeStamp component</p>
-        <button className="like">🤍</button>
+        <p>{body}</p>
+        <p className="entry-time">{currentYears}</p>
+        <button className="like" onClick={()=> toggleLiked(id, !liked)}>{buttonText}</button>
```

Comment:
Since the `toggleLiked` function doesn't do anything other than calling `updateLiked` with the same parameters, you can just call `updateLiked` from here instead of having your own function.